.react-router/types/+routes.ts

```
@@ -0,0 +1,51 @@
+import "react-router"
+
+declare module "react-router" {
+  interface Register {
+    pages: Pages
+    routeFiles: RouteFiles
+  }
+}
+
+type Pages = {
+  "/": {
+    params: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)